setup.cfg

```diff
@@ -64,7 +64,7 @@ install_requires =
     typing_extensions
     toolz>=0.10.0
     tqdm>=4.56.0
-    vispy>=0.6.4, !=0.8.0
+    vispy>=0.9
```

Comment:
can/should we do this before https://github.com/napari/napari/pull/3494?

Reply:
Yeah, we should get the above first. I scanned the PR list but dod not see it. ... Converted to draft.

Reply:
#3494 is in if you want to take this up again?